data/json/monsters/zed_burned.json

```diff
@@ -3,24 +3,24 @@
     "id": "mon_zombie_child_scorched",
     "type": "MONSTER",
     "name": { "str": "zombie kinderling" },
-    "description": "A tiny charred body, jumping and kicked and flailing around in a mockery of playground exercise.  It does not need a face for you to feel bad about killing it.",
+    "description": "A tiny charred body - jumping, kicking, and flailing around in a mockery of playground exercise.  It does not need a face for you to feel bad about killing it.",
     "default_faction": "zombie",
     "bodytype": "human",
-    "species": [ "ZOMBIE" ],
+    "species": [ "ZOMBIE", "HUMAN" ],
     "diff": 2,
     "volume": "30000 ml",
     "weight": "40750 g",
-    "hp": 35,
-    "speed": 90,
+    "hp": 30,
+    "speed": 65,
     "material": [ "flesh" ],
     "symbol": "z",
     "color": "i_brown",
     "aggression": 100,
     "morale": 100,
     "melee_skill": 2,
-    "melee_dice": 1,
-    "melee_dice_sides": 3,
-    "melee_cut": 5,
+    "melee_dice": 2,
+    "melee_dice_sides": 2,
+    "melee_cut": 6,
```

Comment:
I'm not sure why the increase in cutting damage.

Reply:
the damage increase is meant to compensate for the speed decrease as well, the child effectively retains its speed, but losing a small amount of health, and also gains a small amount of damage increase which i figured was better off being distributed among the two types rather than one